ArrayRotation.java

```diff
@@ -0,0 +1,29 @@
+// Java program to rotate an array of length n by
+// k elements
+
+import java.util.Arrays;
+
+public class ArrayRotation {
+    /*Function to left rotate arr[] of length n by k */
+    private int[] rotateArray(int[] arr, int n, int k) {
+        int[] result = new int[arr.length];

```

Comment:
way you write new int[arr.length]; if you have n -???
you should rename var n --> length will be better

Reply:
I already changed n to length